integrationExamples/postbid/oas/creative.html

```diff
@@ -0,0 +1,66 @@
+<!-- this iframe is wrapper for postbid creative, its id has to be unique -->
+<iframe id="unique-prebid-wrapper" src="" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" topmargin="0" leftmargin="0" allowtransparency="true" width="100%" height="100%"></iframe>
```

Comment:
It's probably better to put the unique ID on this iframe in the code (instead of relying on the client to update it).